.github/workflows/build-and-deploy-images.yml

```diff
@@ -5,8 +5,6 @@ on:
   push:
     tags:
       - "v*"
-    branches:
-      - develop
```

Comment:
Just a note to say that we should remove the section regarding developer builds (lines 19-24).  We only every want to build production builds on github actions.

Reply:
got it, thanks!